components/delete_channel_modal/delete_channel_modal.test.tsx

```diff
@@ -69,7 +77,7 @@ describe('components/delete_channel_modal', () => {
         );
 
         wrapper.setState({show: true});
-        wrapper.instance().handleDelete();
+        wrapper.instance().handleDelete(); // Property 'handleDelete' does not exist on type 'Component<{}, {}, any>'
```

Comment:
As noted in the comments, I'm getting a TypeScript error in VSCode: `Property 'handleDelete' does not exist on type 'Component<{}, {}, any>'`.

Reply:
See comment above.